MetaMask/metamask-mobile
# refactor(ramp): order list button labels

## **Description**

This PR refactors de Order list filters to be (All, Purchased, Sold) instead of (All, Buy, Sell).
## **Related issues**

Fixes: #

## **Manual testing steps**

1. Go to Order list in the Activity tab
2. Confirm filter buttons have correct label

## **Screenshots/Recordings**



### **Before**

![image](https://github.com/MetaMask/metamask-mobile/assets/1024246/19dbdde3-9ea1-4414-a83d-4ce22f190dab)

### **After**

<img width="300" src="https://github.com/MetaMask/metamask-mobile/assets/1024246/3dacf4f2-f361-4493-93b7-cf1b8924d6d5" />

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
